src/migrations/1728032163157-dropAllIndexesFKsRelations.ts

```
@@ -0,0 +1,103 @@
+      FROM information_schema.STATISTICS s
+      JOIN information_schema.COLUMNS c
+      ON s.TABLE_NAME = c.TABLE_NAME
+      AND s.TABLE_SCHEMA = c.TABLE_SCHEMA
+      AND s.COLUMN_NAME = c.COLUMN_NAME
+      WHERE s.TABLE_SCHEMA = '${dbName}'
+        AND s.INDEX_NAME != 'PRIMARY'
+      GROUP BY s.TABLE_NAME, s.INDEX_NAME, c.COLUMN_NAME
+    `);
+
+    for (const index of indexesResult) {
+      if (index.EXTRA && index.EXTRA.includes('auto_increment')) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)